Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -197,6 +197,16 @@ RefPtr<WebProcessProxy> WebProcessProxy::processForIdentifier(ProcessIdentifier
     return allProcessMap().get(identifier);
 }
 
+RefPtr<WebProcessProxy> WebProcessProxy::processForConnection(const IPC::Connection& connection)
+{
+    for (Ref webProcessProxy : allProcesses()) {
```

Comment:
WebProcessProxy already implements the type casting traits and there is a similar helper function in AuxiliaryProcessProxy, so it seems like this should just be:

```
return dynamicDowncast<WebProcessProxy>(AuxiliaryProcessProxy::fromConnection(connection));
```

Reply:
Thanks for pointing it out; fixed